pkg/innerring/netmap.go

```diff
@@ -0,0 +1,20 @@
+package innerring
+
+import (
+	"github.com/nspcc-dev/neofs-node/pkg/innerring/processors/netmap/nodevalidation/state"
+	netmapclient "github.com/nspcc-dev/neofs-node/pkg/morph/client/netmap"
+)
+
+/*
+Fle contains dependencies for processor of the Netmap contract's notifications.
+*/
+
+// wraps Netmap contract's client and provides state.NetworkSettings.
+type networkSettings netmapclient.Client
```

Comment:
should it be the client? how about a cache?

Reply:
> should it be the client?

What do you mean?

> how about a cache?

#1835

Reply:
> What do you mean?

current implementation does not allow caching => dont get why we have a separate type if we dont use it

Reply:
Because `netmapclient.Client` does not provide `state.NetworkSettings` by design.